CMakeLists.txt

```diff
@@ -151,9 +151,7 @@ else()
                           FIND_PACKAGE_ARGS)
 
         FetchContent_GetProperties(hwloc_targ)
```

Comment:
GetProperties is redundant, I believe: https://cmake.org/cmake/help/latest/module/FetchContent.html#command:fetchcontent_getproperties